src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -320,6 +323,47 @@ public long StartBackupTask(long taskId, bool isFullBackup)
             return CreateBackupTask(periodicBackup, isFullBackup, SystemTime.UtcNow);
         }
 
+        public async Task Delay(long taskId, TimeSpan? delayInHrs, X509Certificate2 clientCert)
```

Comment:
Can `delayInHrs` be null?
Also, should be `delay`?

Reply:
Done.